tests/e2e/python/chainsaw-test.yaml

```diff
@@ -54,7 +54,7 @@ spec:
         - events:
             namespace: default
         - describe:
-            apiVersion: v1
+            apiVersion: spark.apache.org/v1alpha1
```

Comment:
Why do we need this in this PR, @TQJADE ?

Reply:
I believe this is an orthogonal PR which needs a new JIRA ID.

Reply:
Removed this from this PR